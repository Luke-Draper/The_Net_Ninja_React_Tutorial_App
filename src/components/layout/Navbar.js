import React, { Component } from "react";

class Navbar extends Component {
	render() {
		return (
			<nav className="navbar">
				<h2>Navbar</h2>
			</nav>
		);
	}
}

export default Navbar;
