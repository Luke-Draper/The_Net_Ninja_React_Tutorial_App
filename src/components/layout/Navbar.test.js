import React from "react";
import Navbar from "./Navbar";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("Navbar renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<Navbar />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
