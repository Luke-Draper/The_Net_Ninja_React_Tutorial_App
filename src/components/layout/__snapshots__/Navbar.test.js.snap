// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders without crashing 1`] = `
<nav
  className="nav-wrapper grey darken-3"
>
  <div
    className="container"
  >
    <a
      className="brand-logo"
      href="/"
      onClick={[Function]}
    >
      MarioPlan
    </a>
  </div>
</nav>
`;
